Source/Response.swift

```diff
@@ -42,13 +42,13 @@ public struct DataResponse<Value> {
     public let serializationDuration: TimeInterval
 
     /// The result of response serialization.
-    public let result: Result<Value>
+    public let result: AFResult<Value>
 
     /// Returns the associated value of the result if it is a success, `nil` otherwise.
-    public var value: Value? { return result.value }
+    public var value: Value? { return result.af.value }
```

Comment:
Might we want to get rid of this API and have users pull from the result themselves?

Reply:
If the new intent is to not provide functionality beyond what Apple thought appropriate for Result, then removing this would be inline with that. It depends on if you think providing a shortcut or bandaid over Result is appropriate. 

I'd guess that someone who was unhappy with the Result API would already have their own extension that added `value` to Result and this would be unnecessary.

Reply:
We use this as a convenience all over the place in the test suite. I think it would be unwise for us to make the assumption that the public doesn't need this, b/c they will. The question is just whether we provide it through an `af` namespace, or we require them to extend `Result` themselves. In the best practice mindset, I'd lean towards removing it. In the pragmatic sense, I think everyone else will add it if we don't provide it.

Reply:
That's not what I meant. If we offer public `Result` extensions for accessing the associated values, we don't need to offer these properties on `Response`, since users could just access `response.result.af.value`.